foundations/javascript_basics/loops_and_arrays.md

```diff
@@ -198,8 +198,8 @@ We will teach you the art of actually writing these tests later in the course. F
    - `Copy and sort array`
    - `Shuffle an array`
    - `Filter unique array members`
-1. Follow up by coding along to [JavaScript Array Cardio Practice - Day 1](https://www.youtube.com/watch?v=HB1ZC7czKRs) by Wes Bos. You will need to fork and clone the [JavaScript30 repository](https://github.com/wesbos/JavaScript30) for this.
-1. Watch and code along with [Wes Bos' Array Cardio Day 2](https://www.youtube.com/watch?v=QNmRfyNg1lw).
+1. Follow up by coding along to [JavaScript Array Cardio Practice - Day 1](https://www.youtube.com/watch?v=HB1ZC7czKRs) by Wes Bos. You will need to fork and clone the [JavaScript30 repository](https://github.com/wesbos/JavaScript30) for this. The code for this lesson is in the `04 - Array Cardio Day 1` folder inside the repo. Open the `index-START.html` file and follow along.
+1. Watch and code along with [Wes Bos' Array Cardio Day 2](https://www.youtube.com/watch?v=QNmRfyNg1lw). The code for this lesson is in the `07 - Array Cardio Day 2` folder. Again, open the `index-START.html` file and follow along.
```

Comment:
I find this to be more confusing, even though it is more informative. What do you think about changing the format and instead doing something like this
```
1. Fork and clone Wes Bos' JavaScript30 repository. To follow along with these two exercises, you'll want to open the `index-START.html` file.
    - 04 - Array Cardio Day 1 (link to video)
    - 07 - Array Cardio Day 2 (link to video)
```